runtime/lua/vim/lsp/health.lua

```diff
@@ -33,24 +33,30 @@ local function check_active_clients()
   local clients = vim.lsp.get_clients()
   if next(clients) then
     for _, client in pairs(clients) do
-      local attached_to = table.concat(vim.tbl_keys(client.attached_buffers or {}), ',')
-      report_info(
+      local cmd ---@type string
+      if type(client.config.cmd) == 'table' then
+        cmd = table.concat(client.config.cmd --[[@as table]], ' ')
+      elseif type(client.config.cmd) == 'function' then
+        cmd = tostring(client.config.cmd)
+      end
+      report_info(table.concat({
```

Comment:
giga-nitpick: is it just me or is `report_info` kinda wordy for a local variable? At that point one could just go with the full thing, or shorten it to `info`.